pom.xml

```diff
@@ -13,7 +13,7 @@
     <groupId>com.mabl.integration.jenkins</groupId>
     <artifactId>mabl-integration</artifactId>
     <packaging>hpi</packaging>
-    <version>0.0.11-SNAPSHOT</version>
+    <version>0.0.12-SNAPSHOT</version>
```

Comment:
Actually I think I don't need this change.  It looks like the current version is 0.0.10, so I guess 0.0.11 would be the next version...